sim/hunter/aspects.go

```diff
@@ -120,67 +120,4 @@ func (hunter *Hunter) registerAspectOfTheHawkSpell() {
 	maxRank := hunter.getMaxHawkRank()
 	config := hunter.getAspectOfTheHawkSpellConfig(maxRank)
 	hunter.GetOrRegisterSpell(config)
-}
-
-func (hunter *Hunter) registerAspectOfTheViperSpell() {
```

Comment:
Omg I totally forgot that in vanilla they only have Viper Sting. Vanilla is wild 😂

Reply:
shut up and pull green